### ISSUE TITLE
Feature: Video support 

### ISSUE THREAD

**Issue by zombiestruck** [PhaserEditor2D/PhaserEditor2D-v3#246]
It would be very handy to be able to add videos to a scene.

**Comment by ChildLearningClub** [comment]
+ 1

**Comment by zombiestruck** [comment]
Also it may be nice to have a video queue with an option to loop the queue. I'm using the video on complete event to chain them.

**Comment by PhaserEditor2D** [comment]
Hi @zombiestruck!

> Also it may be nice to have a video queue with an option to loop the queue. I'm using the video on complete event to chain them.

Please can you explain this better? The "video queue" is something supported by Phaser?

**Comment by ChildLearningClub** [comment]
I think maybe just the “on video complete” signal is fired and @zombiestruck has another video play after and does that for a few videos. So in PhaserEditor2D to have on the right hand side after say a video “key” or url box there could be a plus button to add more that would then chain them together. Additionally there could be a “timer” timer wait time in seconds or whatever before starting the next video asset or url linked video. Correct me if I’m wrong @zombiestruck ?

**Comment by PhaserEditor2D** [comment]
It looks like something too specific. I think you can add the videos to an ObjectList and later execute them using a utility you did manually. Something like:

```
manager = new VideoManager(this.videoList)
manager.play()
```

**Comment by ChildLearningClub** [comment]
I thought maybe that was the case too.

**Comment by zombiestruck** [comment]
```
attractVid1.on('complete', function(video){

```

**Comment by zombiestruck** [comment]
```
attractVid1.on('complete', function(video){

```
